app/controllers/attended_events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class AttendedEventsController < ApplicationController
+  before_action :logged_in_user, only: [:create]
+  before_action :check_subscription_status, only: [:create]
+
+  
+  def create
+    @user = current_user().id
+    @attend = AttendedEventsTbl.new(attendee_id: @user, attended_event_id: params[:event])
+    @attend.save
+    flash[:success] = "Succesfully subscribed to the event."
+    redirect_to "/users/#{@user}"
+  end
+
+  private 
+    def logged_in_user
+      unless logged_in?
+        flash[:danger] = "Please log in."
+        redirect_to signin_url
+      end
+    end
+
+    def check_subscription_status
```

Comment:
this method should be used in the view to show/hide attend button, please move it to appropriate controller.

Reply:
sent method to EVENT helper file and used it in 'show' page of the event